de-identification-entry-spark/src/main/java/com/ibm/whc/deid/external/spark/udf/DeIdSynapseUDF.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2022
+ * © Merative US L.P. 2022
```

Comment:
Is this the official copyright notice?  Do we need non-ascii character(©) here?

Reply:
Yes - the notice specifically asked for the © character due to translation issues if you use (c) or copyright in some countries.